### PR TITLE
intezer: add polling commands

### PR DESCRIPTION
**Pull request by matany90**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
- Add polling option to file, url and endpoint analysis commands
- Replace generic polling with polling command in Playbook

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [x] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by matany90** [comment]
@yaakovpraisler the build pass, can you please review?

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2.py [hunk 1]
```diff
@@ -656,6 +801,46 @@ def _refine_gene_counts(summary: dict):
...
 
 
+def _get_file_analysis_options(args: dict, intezer_api: IntezerApi) -> dict:
+    file_id = args.get('file_entry_id')
+    file_data = demisto.getFilePath(file_id)
+
+    zip_password = args.get('zip_password')
+    disable_dynamic_execution = args.get('disable_dynamic_execution')
+    disable_static_extraction = args.get('disable_static_extraction')
+    sandbox_command_line_arguments = args.get('sandbox_command_line_arguments')
```
Comment (yaakovpraisler) [inline Packs/Intezer/Integrations/IntezerV2/IntezerV2.py:811]: I can't find the `sandbox_command_line_arguments` argument in the yaml file

File: Packs/Intezer/Playbooks/playbook-Intezer_-_Analyze_File_and_URL.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 id: Intezer - Analyze File and URL
 version: -1
-fromversion: 6.5.0
+fromversion: 6.2.0
```
Comment (yaakovpraisler) [inline Packs/Intezer/Playbooks/playbook-Intezer_-_Analyze_File_and_URL.yml:3]: @efelmandar - Do we support this?

File: Packs/Intezer/ReleaseNotes/1_5_0.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+##### Intezer v2
+- Support polling using schedule tasks instead of generic polling
```
Comment (yaakovpraisler) [inline Packs/Intezer/ReleaseNotes/1_5_0.md:4]:
```suggestion
- Added support for *polling* to the following commands:
 - ***intezer-analyze-by-hash***
 - ***intezer-analyze-url***
 - ***intezer-analyze-by-file***
- Added the following new *polling* commands:
 - ***intezer-get-file-analysis-result***
 - ***intezer-get-url-analysis-result***
 - ***intezer-get-endpoint-analysis-result***
```

**Review by yaakovpraisler — CHANGES_REQUESTED** [review]
Hi @matany90, Thank you for your contribution! Nice work :)

Just wanted to verify - Did you added the new result commands only to support polling in the existing commands (I ask because you can support this without adding new commands)? or you still want the result commands regardless of the existing commands?

@efelmandar - Please approve the playbook changes

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2.py [hunk 4]
```diff
@@ -131,7 +147,7 @@ def check_is_available(intezer_api: IntezerApi, args: dict) -> str:
         return f'Error connecting to Analyze Base url.\n{error}'
 
 
-def analyze_by_hash_command(intezer_api: IntezerApi, args: Dict[str, str]) -> CommandResults:
+def analyze_by_hash_command(args: Dict[str, str], intezer_api: IntezerApi) -> PollResult:
```
[the remark this answers is not in the document]
Comment (yaakovpraisler) [reply Packs/Intezer/Integrations/IntezerV2/IntezerV2.py:150]: Same for the `analyze_url_command` and `analyze_by_uploaded_file_command` command functions

**Comment by matany90** [comment]
@yaakovpraisler
Thank you for the review!

About your first question about the new commands:
I still want to add those three new commands, regardless of the polling functionality I added in "intezer-analyze-by-hash", "intezer-analyze-by-file" and "intezer-analyze-url" commands.

About the Playbook version that I changed from 6.5.0 to 6.2.0, I did it by mistake; I changed it back to 6.5.0

The rest of the issues you raised were fixed :)

File: Packs/Intezer/Playbooks/playbook-Intezer_-_Analyze_File_and_URL.yml (hunk 2 above)
Comment (matany90) [reply Packs/Intezer/Playbooks/playbook-Intezer_-_Analyze_File_and_URL.yml]: Probably my mistake; I changed it back to 6.5.0

**Review by yaakovpraisler — COMMENTED** [review]
Looks good! Just a few small comments -

Please let me know when you're available so I'll schedule us a call for a demo
@kgal-pan FYI

**Comment by yaakovpraisler** [comment]
Hi @matany90 -
We can do it on Thursday, just let me know when it's convenient for you and I'll schedule us a call.

**Comment by matany90** [comment]
@yaakovpraisler 
Thursday around 11:30 works for me

**Comment by efelmandar** [comment]
Hi @matany90 and thank you for your contribution, I reviewed the changes and all looks good, great work!

**Review by yaakovpraisler — APPROVED** [review]

**Pull request merged**